contracts/abstract/JBProject.sol

```diff
@@ -29,7 +29,7 @@ abstract contract JBProject is Ownable {
   }
 
   /** 
-      Received funds go streight to the project.
+      Received funds go straight to the project.
     */
   receive() external payable {
     _pay(msg.sender, '', false, address(0));
```

Comment:
i think the token representation of ETH is no longer 0x00. 

https://github.com/jbx-protocol/juice-contracts-v2/blob/81c78393a4ff46dfcd354beeaa771f2516b2a7ad/contracts/JBETHPaymentTerminal.sol#L92